standalone/header-footer/src/StandaloneProvider/StandaloneProvider.js

```diff
@@ -20,7 +20,9 @@ export default class StandaloneProvider extends Component {
   }
 
   updateProps(componentProps) {
-    this.setState({ componentProps });
+    this.state.componentProps = { ...this.state.componentProps, ...componentProps };
```

Comment:
We shouldn't be editing state directly like this. Why is the call to set state not working correctly?